### PR TITLE
 fix(FEC-7344, FEC-7257): update initial player poster

### PR DESCRIPTION
**Pull request by dvh91**
### Description of the Changes

Update initial player poster at `source_selected` event

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/engine-connector/engine-connector.js [hunk 1]
```diff
@@ -59,10 +59,14 @@ class EngineConnector extends BaseComponent {
       this.props.updateDuration(this.player.duration);
     });
 
+    this.player.addEventListener(this.player.Event.SOURCE_SELECTED, () => {
+      // TODO: change when player will expose the player.poster at this stage
+      this.props.updatePlayerPoster(this.player.config.metadata.poster);
```
Comment (OrenMe) [inline src/components/engine-connector/engine-connector.js]: @dvh91 `player.poster` is already exposed@https://github.com/kaltura/playkit-js/blob/master/src/player.js#L632

Comment (dvh91) [reply src/components/engine-connector/engine-connector.js]: @OrenMe not at the stage of `SOURCE_SELECTED`.

**Comment by dan-ziv** [comment]
@OrenMe LGTM now
Please review again

File: src/components/pre-playback-play-overlay/pre-playback-play-overlay.js [hunk 2]
```diff
@@ -125,8 +125,12 @@ class PrePlaybackPlayOverlay extends BaseComponent {
     if ((!props.isEnded && !props.prePlayback) || (!props.isEnded && this.autoplay)) {
       return undefined;
     }
+    let rootStyle = {
+      backgroundColor: '#000',
```
Comment (OrenMe) [inline src/components/pre-playback-play-overlay/pre-playback-play-overlay.js]: why do we inline the style? we wouldn't be able to easily override it later.

**Pull request merged**